javascript/javascript1/week3/homework4.js

```diff
@@ -0,0 +1,63 @@
+// Smart-ease - Back to the basics!
+// NOnoN0nOYes (Note taking app)
+
+// The saveNote function should push an object to the notes array with the keys content and id.
+
+/**
+ *
+ *
+ * @param {string} content
+ * @param {number} id
+ */
+
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id,
+  });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+//unique feature of colored log
+console.log("\x1b[31m%s\x1b[0m", "\n\t SaveNote task \n");
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id) {
+  let result = "";
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id) {
+      return (result = notes[i]);
+    }
+  }
+}
+
+const firstNote = getNote(1);
+
+console.log("\x1b[34m%s\x1b[0m", "\t Get a note task \n");
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+// Log out notes
+
+function logOutNotesFormatted() {
+  answer = [];
+  for (let i = 0; i < notes.length; i++) {
+    answer.push(
+      `The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}`
+    );
+  }
+  return console.log(answer.join("\n"));
```

Comment:
As before here I would actually just console.log out each string without using arrays. Having said that it isn't incorrect and great you are feeling comfortable with arrays!